.sling-module.json

```diff
@@ -1,6 +1,6 @@
 {
     "jenkins": {
-        "jdks": [8,11],
+        "jdks": [11,17,21],
```

Comment:
This is the default, I would only add values which deviate from it, i.e. remove element "jdks" completely